src/globus_cli/commands/search/index/create.py

```diff
@@ -14,7 +14,7 @@
 def create_command(
     login_manager: LoginManager, *, display_name: str, description: str
 ) -> None:
-    """Create a new Index"""
+    """Create a new Index."""
```

Comment:
If our standard for groups, flows, and timers are lowercase, should this be lowercase as well?

Reply:
Yeah, I would like us to lowercase these simply for consistency's sake. (I don't feel very strongly about what convention we use, so long as I have some kind of guideline to follow.)

I'll apply the changes you've suggested on this front, but I want to avoid trying to enforce that universally in the scope of this PR. IMO including these changes is a little bit of (acceptable) scope creep but shouldn't be a blocker for getting this PR merged. We may need another pass over these anyway -- I want to see if `click` can support boldface ANSI codes in helptext, which is the styling we're using on the docs site.